prow/cmd/hook/main.go

```diff
@@ -18,6 +18,7 @@ package main
 
 import (
 	"flag"
+	"k8s.io/test-infra/prow/rocketchat"
```

Comment:
nit: pls move these down to other k8s.io/t-i imports

Reply:
done